pyintegration/deephaven/filters.py

```diff
@@ -0,0 +1,43 @@
+#
+#   Copyright (c) 2016-2022 Deephaven Data Labs and Patent Pending
+#
+
+#
+#   Copyright (c) 2016-2022 Deephaven Data Labs and Patent Pending
+#
+""" This module implement various filters that can be used in deephaven table's filter operations."""
```

Comment:
I really think the design of this library is going to start paying dividends. I like this.